main.cpp

```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: knife
+//  action 1: Stab a Player
+knife.stab();
+//  action 2: Slash a Player
+knife.slash();
+//  action 3: Hold knife Menacingly
+knife.menace();

```

Comment:
the player holds the knife menacingly.
pick a different action and revise the pseudo-code
- [ ] revise